app/src/main/java/com/github/se/wanderpals/ui/screens/suggestion/CommentOptionsBottomSheet.kt

```diff
@@ -81,26 +81,28 @@ fun CommentBottomSheet(
                       }
                     }
 
-                Box(
-                    modifier =
-                        Modifier.fillMaxWidth()
-                            .clickable(
-                                onClick = {
-                                  viewModel.editCommentOption()
-                                  onEdit(selectedComment!!.text)
-                                })
-                            .padding(16.dp)
-                            .testTag("editCommentOption"),
-                    contentAlignment = Alignment.CenterStart) {
-                      Row(verticalAlignment = Alignment.CenterVertically) {
-                        Icon(
-                            imageVector = Icons.Outlined.Create,
-                            contentDescription = "Edit",
-                            modifier = Modifier.size(24.dp))
-                        Spacer(modifier = Modifier.width(16.dp)) // Space between icon and text
-                        Text("Edit comment", style = MaterialTheme.typography.bodyLarge)
+                if (SessionManager.getIsNetworkAvailable()) {
```

Comment:
We can use just disabled the possibility to click on the button by adding: 
enabled = SessionManager.getIsNetworkAvailable() and avoid to increase cognitive complexitywith the if

Reply:
Unfortunately since it's a clickable modifier, no composable has an enable option

Reply:
.clickable has an enable field

.clickable(
enabled = ...
onlick{})

Reply:
Didn't know, will do, thx